mage_ai/data_cleaner/shared/utils.py

```diff
@@ -54,7 +59,11 @@ def clean_series(series, column_type, dropna=True):
 
 
 def clean_dataframe(df, column_types, dropna=True):
-    return df.apply(lambda col: clean_series(col, column_types[col.name], dropna=dropna), axis=0)
+    cols = [df[col] for col in df.columns]
+    ctypes = [column_types[col] for col in df.columns]
+    dropnas = [dropna for _ in df.columns]
+    clean_cols = run_parallel_multiple_args(clean_series, cols, ctypes, dropnas)
```

Comment:
does it help improve performance? how much does it improve?

Reply:
I tested on a dataset with around 10000 rows (MovieLens dataset), each of them containing a list in string form. Initially this took around 6.4 seconds, but when multithreaded this reduced to about 5.6 seconds. Not too big of an improvement but it was somewhat better so I kept it in.